src/qml/components/Footer.qml

```diff
@@ -5,12 +5,13 @@ import QtQuick.Controls 2.12
 import "../theme"
 
 Item {
+    property string _text: ""
```

Comment:
If you use footer with empty text in this task, prefer to hide footer because it occupies space

Reply:
Made it unvisible